src/utils/barcode-scan.js

```diff
@@ -1,5 +1,13 @@
 import { BrowserPDF417Reader } from '@zxing/browser';
 
+/*
+ * Reads a scanned Image file, and returns the raw image data
+ *
+ * @memberof utils
+ * @function readImageFile
+ * @params {object} file - image to be read
+ * @returns {Promise} Promise - returns a Promise with base64 image
+ */
```

Comment:
Can add a new line between JSDoc comment and function to keep things consistent.